release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -90,8 +90,101 @@ This release adds improvements related to the following components and concepts.
 {product-title} 3.9 introduces the following notable technical changes.
 
 [discrete]
-[[ocp-310-change1]]
-=== Change 1
+[[ocp-310-major-changes-to-cluster-architecture]]
+=== Major Changes to Cluster Architecture
+
+{product-title} 3.10 introduces major architecture changes in how control
+plane and node components are deployed, affecting new installations and upgrades
+from {product-title} 3.9.
+
+_(New architecture changes diagram, WIP)_
+
+The following sections highlight the most significant changes, with more detail
+provided in the Architecture Guide.
+
+[discrete]
+[[ocp-310-control-plane-changes]]
+==== Control Plane as Static Pods
+
+While previously run as systemd services, the control plane components (etcd,
```

Comment:
etcd maybe not 100% become a static pod, this depends on how etcd host is defined, if etcd co-locate with master, then etcd would be static pod, if etcd is a standalone host, there is no etcd static pod, just a common etcd rpm install.